jest/transform.js

```diff
@@ -38,10 +63,8 @@ module.exports.transform = Object.fromEntries(
             TS_JEST_PATH,
             {
               ...value[1],
-              isolatedModules:
-                value[1]?.isolatedModules ??
```

Comment:
Removed some of this complexity because `defaults.transform` doesn't populate `isolatedModules` nor `tsconfig`. We can always update this in future if that changes.